source/personal-growth/coaching-and-one-on-ones.html.md

```diff
@@ -52,8 +52,7 @@ questions you can ask yourself to prepare for a 1:1:
 
 During the 1:1, your manager will take notes on what you discuss and what you commit to, so that you
 can review your progress in the following 1:1. This creates a sense of accountability and ensures no
-stone is left unturned. You will get the meeting minutes via email or Slack, so that you can
-reference them later.
+stone is left unturned. You will be able to review the meeting minutes via [Lattice](https://lattice.com/) or Slack, so that you can reference them later.
```

Comment:
@nahemaandrade the change is good but I would prefer if you break the line to be in length similar to the overall average of the line length of the page. In this case you could send `or Slack` and on to the next line.

Reply:
The update was made.